dist/milo_ui.bundle.js

```diff
@@ -2335,9 +2336,11 @@ function MLTime_set(value) {
         return;
     }
 
+    var hours = this.utc ? time.getUTCHours() : time.getHours();
```

Comment:
If `utc` is enabled, the element will be rendered using the utc hours/minutes rather than the local time